docs/Cask-Cookbook.md

```diff
@@ -113,6 +113,9 @@ Each of the following stanzas is required for every cask.
 | [`name`](#stanza-name)       | yes                           | String providing the full and proper name defined by the vendor. |
 | [`desc`](#stanza-desc)       | no                            | One-line description of the cask. Shown when running `brew info`. |
 | `homepage`                   | no                            | Application homepage; used for the `brew home` command. |
+| [`livecheck`](#stanza-livecheck)           | no                            | Ruby block describing how to find updates for this cask. Supersedes `appcast`. |
+| [`depends_on`](#stanza-depends_on)         | yes                           | List of dependencies and requirements for this cask. |
+| [`zap`](#stanza-zap)                       | yes                           | Additional procedures for a more complete uninstall, including user files and shared resources. |
```

Comment:
```suggestion
| `homepage`                   | no                            | Application homepage; used for the `brew home` command. |
| [`livecheck`](#stanza-livecheck)   | no                      | Ruby block describing how to find updates for this cask. Supersedes `appcast`. |
| [`depends_on`](#stanza-depends_on) | yes                     | List of dependencies and requirements for this cask. |
| [`zap`](#stanza-zap)               | yes                     | Additional procedures for a more complete uninstall, including user files and shared resources. |
```

(suggestion basically attempts to fix the spacing to align the table `|`'s again)